app/views/RoomInfoView/index.tsx

```diff
@@ -245,6 +247,17 @@ class RoomInfoView extends React.Component<IRoomInfoViewProps, IRoomInfoViewStat
 			})
 		);
 
+	setUser = async (user: IUser) => {
+		const { usersRoles } = this.props;
+		const userRoles = usersRoles.find(u => u?.username === user.username);
```

Comment:
Considering the error found, maybe here you still need to test if the user exists on the redux, if there isn't still a need to request as before.